--- origsrc/ming-0.4.2/configure.in	2008-09-17 19:44:39.000000000 -0500
+++ src/ming-0.4.2/configure.in	2009-04-13 22:21:18.588875000 -0500
@@ -159,9 +159,10 @@ $HOME/lib \
 /usr/lib"
 
 	for i in $tcllibdirs_default; do
-	  for suf in 8.3 8.4 ""; do
-	    if test -f $i/libtcl$suf.so || test -f $i/libtcl$suf.a; then
+	  for suf in 8.5 8.4 8.3 ""; do
+	    if test -f $i/libtcl$suf.dll.a || test -f $i/libtcl$suf.a; then
 	     TCL_LIB_DIR=$i
+	     TCL_LIB_SPEC="-L$i -ltcl$suf"
 	     break
 	    fi
 	  done
@@ -175,6 +176,11 @@ tclincdirs_default="\
 /usr/include/tcl-private/generic \
 /usr/include/tk-private/generic \
 /usr/include/itcl-private/generic \
+/usr/include/tcl8.5 \
+/usr/include/tcl8.5/tcl-private/generic \
+/usr/include/tcl8.5/tk-private/generic \
+/usr/include/tcl8.5/itcl-private/generic \
+/usr/include/tcl8.5 \
 /usr/include/tcl8.4 \
 /usr/include/tcl8.4/tcl-private/generic \
 /usr/include/tcl8.4/tk-private/generic \
@@ -209,6 +215,7 @@ fi
 
 AM_CONDITIONAL(BUILD_TCL_EXTENSION, test x"$tcl_ext" = xyes)
 AC_SUBST(TCL_LIB_DIR)
+AC_SUBST(TCL_LIB_SPEC)
 AC_SUBST(TCL_INC_DIR)
 
 dnl --------------------------------------------
--- origsrc/ming-0.4.2/tcl_ext/Makefile.am	2008-05-19 09:06:19.000000000 -0500
+++ src/ming-0.4.2/tcl_ext/Makefile.am	2009-04-13 22:21:55.120125000 -0500
@@ -6,7 +6,7 @@ mingc_ladir  = $(libdir)/ming/tcl
 mingc_la_SOURCES  = ming_wrap.c
 mingc_la_CPPFLAGS  = -I.. -I../src -I$(TCL_INC_DIR)
 mingc_la_LDFLAGS  = -module -no-undefined -avoid-version
-mingc_la_LIBADD  = ../src/libming.la
+mingc_la_LIBADD  = ../src/libming.la $(TCL_LIB_SPEC)
 
 MAINTAINERCLEANFILES = ming_wrap.c
 
