--- origsrc/ming-0.4.4/perl_ext/Makefile.PL	2011-10-26 08:33:18.000000000 +0200
+++ src/ming-0.4.4/perl_ext/Makefile.PL	2011-11-05 15:33:25.046875000 +0100
@@ -68,7 +68,7 @@ my @SWF_MODULES_NAMES = (
 );
 
 my @XS_OBJECTS = map "${BUILDDIR}/$_", @SWF_MODULES_NAMES;
-push @XS_OBJECTS, "${SRCDIR}/swf_util";
+push @XS_OBJECTS, "${BUILDDIR}/swf_util";
 push @XS_OBJECTS, "${BUILDDIR}/SWF";
 my $XS_OBJECTS = join ('$(OBJ_EXT) ' , (@XS_OBJECTS)) . '$(OBJ_EXT)';
