--- origsrc/ming-0.4.0.beta5/Makefile.am	2007-09-17 06:35:31.000000000 -0500
+++ src/ming-0.4.0.beta5/Makefile.am	2008-03-18 09:07:40.640625000 -0500
@@ -9,6 +9,7 @@
 # requirement to 1.4d, but we didn't check it
 #
 AUTOMAKE_OPTIONS = foreign dist-bzip2 dist-zip 1.4d
+ACLOCAL_AMFLAGS = -I macros
 
 additionnal_dist_subdirs = 
 
--- origsrc/ming-0.4.0.beta5/configure.in	2007-09-19 12:43:51.000000000 -0500
+++ src/ming-0.4.0.beta5/configure.in	2008-03-18 09:07:40.906250000 -0500
@@ -146,6 +146,7 @@
 	  for suf in 8.3 8.4 ""; do
 	    if test -f $i/libtcl$suf.so || test -f $i/libtcl$suf.a; then
 	     TCL_LIB_DIR=$i
+	     TCL_LIB_SPEC="-L$i -ltcl$suf"
 	     break
 	    fi
 	  done
@@ -193,6 +194,7 @@
 
 AM_CONDITIONAL(BUILD_TCL_EXTENSION, test x"$tcl_ext" = xyes)
 AC_SUBST(TCL_LIB_DIR)
+AC_SUBST(TCL_LIB_SPEC)
 AC_SUBST(TCL_INC_DIR)
 
 dnl --------------------------------------------
--- origsrc/ming-0.4.0.beta5/perl_ext/Makefile.PL	2006-12-15 09:54:39.000000000 -0600
+++ src/ming-0.4.0.beta5/perl_ext/Makefile.PL	2008-05-10 23:25:38.796875000 -0500
@@ -94,7 +94,7 @@
     clean   => {
         FILES   => "@do_clean",
     },
-    'MYEXTLIB'     => $MING_DIR . '/src/.libs/libming.a',
+    'MYEXTLIB'     => $MING_DIR . '/src/.libs/libming.dll.a',
     'OBJECT'       => "$XS_OBJECTS",
     #'LIBS'         => [ "-lz" ],
 		'LIBS'         => [ $libs ],
@@ -104,9 +104,9 @@
 	my $self = shift;
 	write_myconfig_file($self->{INSTALLSITEARCH});
 
-	for (qw(ming.h libming.a)){
-		$self->{PM}->{"$MING_DIR/$_"} = '$(INST_ARCHLIB)' . "/auto/SWF/include/$_";
-	}
+#	for (qw(ming.h libming.a)){
+#		$self->{PM}->{"$MING_DIR/$_"} = '$(INST_ARCHLIB)' . "/auto/SWF/include/$_";
+#	}
 	$self->{PM}->{'perl_swf.h'} = '$(INST_ARCHLIB)' . "/auto/SWF/include/perl_swf.h";
 	return $self->MM::install;
 }
--- origsrc/ming-0.4.0.beta5/perl_ext/Makefile.am	2006-12-15 09:54:39.000000000 -0600
+++ src/ming-0.4.0.beta5/perl_ext/Makefile.am	2008-03-18 09:07:40.921875000 -0500
@@ -25,7 +25,7 @@
 	$(MAKE) -f Makefile.perl install
 
 Makefile.perl: Makefile.PL
-	srcdir=$(srcdir) builddir=$(builddir) $(PERL) $< MAKEFILE=Makefile.perl
+	$(PERL) $< MAKEFILE=Makefile.perl INSTALLDIRS=vendor
 
 clean-local: Makefile.perl
 	$(MAKE) -f Makefile.perl realclean
--- origsrc/ming-0.4.0.beta5/tcl_ext/Makefile.am	2006-12-15 09:54:39.000000000 -0600
+++ src/ming-0.4.0.beta5/tcl_ext/Makefile.am	2008-03-18 09:07:40.921875000 -0500
@@ -6,7 +6,7 @@
 mingc_la_SOURCES  = ming_wrap.c
 mingc_la_CPPFLAGS  = -I.. -I../src -I$(TCL_INC_DIR)
 mingc_la_LDFLAGS  = -module -no-undefined -avoid-version
-mingc_la_LIBADD  = ../src/libming.la
+mingc_la_LIBADD  = ../src/libming.la $(TCL_LIB_SPEC)
 
 MAINTAINERCLEANFILES = ming_wrap.c
 
--- origsrc/ming-0.4.0.beta5/util/outputtxt.c	2007-09-17 19:16:48.000000000 -0500
+++ src/ming-0.4.0.beta5/util/outputtxt.c	2008-03-18 09:07:40.937500000 -0500
@@ -129,7 +129,7 @@
 
 static unsigned INDENT=0;
 
-static void
+static int
 iprintf(const char* fmt, ...)
 {
 	va_list ap;
